fmpz_mod_mat.h

```diff
@@ -214,7 +214,11 @@ FLINT_DLL void fmpz_mod_mat_mul(fmpz_mod_mat_t C, const fmpz_mod_mat_t A, const
 FLINT_DLL void _fmpz_mod_mat_mul_classical_threaded_pool_op(fmpz_mod_mat_t D,
       const fmpz_mod_mat_t C, const fmpz_mod_mat_t A, const fmpz_mod_mat_t B,
                       int op, thread_pool_handle * threads, slong num_threads);
- 
+
+FLINT_DLL void _fmpz_mod_mat_mul_classical_threaded(fmpz_mod_mat_t D,
```

Comment:
This function doesn't exist, as far as I can tell. It can be removed from both docs and header.